ui/src/services/NodeVolumesUtils.test.js

```diff
@@ -420,3 +422,96 @@ it('should return error when called with failed Ready condition', () => {
 });
 
 // }}}
+
+// getVolumeListData
+const props_volume_filtered_by_node_master0 = {
+  history: {
+    length: 12,
+    action: 'POP',
+    location: {
+      pathname: '/volumes/prom-m0-reldev',
+      search: '?node=master-0',
+      hash: '',
+      key: 'u757uz',
+    },
+  },
+  location: {
+    pathname: '/volumes/prom-m0-reldev',
+    search: '?node=master-0',
+    hash: '',
+    key: 'u757uz',
+  },
+  match: {
+    path: '/volumes',
+    url: '/volumes',
+    isExact: false,
+    params: {},
+  },
+};
```

Comment:
Do we really need all these props? It seems we only read the `location` part anyway...

On the side, it seems to me there are way too many things happening in this selector, and maintaining this code and associated tests will be a pain when the behaviour changes. We'll see.